backend/apps/mentorship/api/internal/queries/module.py

```diff
@@ -0,0 +1,50 @@
+"""OWASP module GraphQL queries."""
+
+import logging
+
+import strawberry
+from django.core.exceptions import ObjectDoesNotExist
+
+from apps.mentorship.api.internal.nodes.module import ModuleNode
+from apps.mentorship.models import Module
+
+logger = logging.getLogger(__name__)
+
+
+@strawberry.type
+class ModuleQuery:
+    """Module queries."""
+
+    @strawberry.field
+    def get_program_modules(self, program_key: str) -> list[ModuleNode]:
+        """Get all modules by program Key. Returns an empty list if program is not found."""
+        return (
+            Module.objects.filter(program__key=program_key)
+            .select_related("program", "project")
+            .prefetch_related("mentors__github_user")
+            .order_by("started_at")
+        )
+
+    @strawberry.field
+    def modules_by_project(self, project_key: str) -> list[ModuleNode]:
```

Comment:
Use consistent naming
```suggestion
    def get_project_modules(self, project_key: str) -> list[ModuleNode]:
```